cmd/helpers.go

```diff
@@ -90,7 +215,7 @@ func BuildApplication(binaryName string, forceRebuild bool, buildMode string, pa
 	}
 
 	buildCommand := slicer.String()
-	buildCommand.Add("mewn")
+	buildCommand.Add("go")
```

Comment:
The `wails serve` command now fails on my machine.

To make it work, I had to move the lines 220-223 (add the `-i` flag) after the line 225 `buildCommand.Add("build")`

Reply:
Thanks for the pointers!  I hadn't validated that `wails serve` was working yet.